Tests/Shared/SampleData/CustomAttributesSampleData.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+
+public enum CustomAttributesSampleData {
```

Comment:
CustomAttributesSampleData - is this for device attributes / profile attributes or just a set of data for any scenario requiring user data ?

Reply:
Set of samples for *all* places we accept custom attributes - identify, track, profile, device. 

I'll add a comment to the file saying that. It would be helpful.